boards/shields/st7735r_generic/Kconfig.defconfig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_ST7735R_ADA_160x128
```

Comment:
_ADA_? Is it not a generic one?

Reply:
It seems like other shields, like e.g. the shield-folder for the 'sister-driver' st7789v_generic or the waveshare_epaper, do often group overlays. I thus see no problem in defining an overlay for exactly the ADA-variant I have access to.
The use of the word 'generic' might be misleading though. I'v just followed the naming used for st7789v, but looking at the commit history for that it was maybe the intention in the beginning to make it generic. I'll rename the shield folder